src/main/java/net/glowstone/block/itemtype/ItemType.java

```diff
@@ -164,10 +166,67 @@ public final String toString() {
         return getClass().getSimpleName() + "{" + getId() + " -> " + getMaterial() + "}";
     }
     
+    //Code for wearable items
+    private ItemWearablePosition wearablePosition = ItemWearablePosition.NON_WEARABLE;
+    private int armorPoints = 0;
+    private boolean fastEquip = false;
+    
     public ItemWearablePosition getWearablePosition() {
         return wearablePosition;
     }
     public void setWearablePosition(ItemWearablePosition position) {
```

Comment:
I removed it because Skull and Pumpkin had different classes that both extended from different classes, so simply extending them from ItemWearable wouldn't work